rclpy/src/rclpy/context.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLPY__CONTEXT_HPP_
+#define RCLPY__CONTEXT_HPP_
+
+#include <Python.h>
+#include <pybind11/pybind11.h>
+#include <rcl/context.h>
+#include <rcl/types.h>
+
+#include <stdexcept>
+
+extern "C" {
+#include "rclpy_common/handle.h"
+}
+
+
```

Comment:
Is this double-newline meaningful?

Reply:
Habit from Python PEP 8 requiring double line breaks between definitions in module scope. Removed extra newline in da5edcc